dfetch/manifest/manifest.py

```diff
@@ -172,6 +172,16 @@ def projects(self) -> Sequence[ProjectEntry]:
         """Get a list of Projects from the manifest."""
         return list(self._projects.values())
 
+    def selected_projects(self, names: Sequence[str]) -> Sequence[ProjectEntry]:
+        """Get a list of Projects from the manifest with the given names."""
+        return (
+            list(
+                project for project in self._projects.values() if project.name in names
+            )
+            if names
```

Comment:
This doesn't seem in place after the return.

Reply:
It is a "pythonic"  contstruction, kind of a ternary ==>

if names is given then return a filtered list otherwise return the unfiltered list

Reply:
Ah, now I understand. My C brain made a short circuit at this statement.